### DISCUSSION TITLE
高亮自定义关键字（正则表达式）

### DISCUSSION THREAD

**Discussion by jayyah** [electerm/electerm#2772] [General]
我看下历史issues，好像20年就有人提出，但是一直没有这个功能，什么时候可以实现嘛？

**Comment by zxdong262** [comment]
我记得好像研究一下有点麻烦就搁置了，后来就被更多的issue淹没了，个人项目，没有啥路线图，维护能力不足，更倾向于处理简单的问题。。。

**Reply by jayyah** [reply]
好吧，辛苦了
